[Errno 5] Input/output error
<class 'OSError'>
[Errno 5] Input/output error
<traceback object at 0xffff93dbd780>
<frame at 0xffff95166640, file 'main.py', line 51, code <module>>
156
42

[Errno 5] Input/output error
<class 'OSError'>
[Errno 5] Input/output error
<traceback object at 0xffff93da04c0>
<frame at 0xffff95166640, file 'main.py', line 51, code <module>>
156
42

[Errno 5] Input/output error
<class 'OSError'>
[Errno 5] Input/output error
<traceback object at 0xffff93f29740>
<frame at 0xffff95166640, file 'main.py', line 51, code <module>>
156
42

[Errno 5] Input/output error
<class 'OSError'>
[Errno 5] Input/output error
<traceback object at 0xffff93f38b00>
<frame at 0xffff95166640, file 'main.py', line 51, code <module>>
156
42

[Errno 5] Input/output error
<class 'OSError'>
[Errno 5] Input/output error
<traceback object at 0xffff93f6b600>
<frame at 0xffff95166640, file 'main.py', line 51, code <module>>
156
42

[Errno 5] Input/output error
<class 'OSError'>
[Errno 5] Input/output error
<traceback object at 0xffff93f0c200>
<frame at 0xffff95166640, file 'main.py', line 51, code <module>>
156
42

[Errno 5] Input/output error
<class 'OSError'>
[Errno 5] Input/output error
<traceback object at 0xffff93de7340>
<frame at 0xffff95166640, file 'main.py', line 51, code <module>>
156
42

[Errno 5] Input/output error
<class 'OSError'>
[Errno 5] Input/output error
<traceback object at 0xffff9410ad40>
<frame at 0xffff95166640, file 'main.py', line 51, code <module>>
156
42

[Errno 5] Input/output error
<class 'OSError'>
[Errno 5] Input/output error
<traceback object at 0xffff93db5700>
<frame at 0xffff95166640, file 'main.py', line 51, code <module>>
156
42

[Errno 5] Input/output error
<class 'OSError'>
[Errno 5] Input/output error
<traceback object at 0xffff93f82300>
<frame at 0xffff95166640, file 'main.py', line 51, code <module>>
156
42

